util/dvsim/SimCfg.py

```diff
@@ -547,8 +547,8 @@ def _cov_unr(self):
         coverage exclusions.
         '''
         # TODO, Only support VCS
-        if self.tool != 'vcs':
-            log.error("Currently only support VCS for coverage UNR")
+        if self.tool not in ['vcs', 'xcelium']:
+            log.error("Currently only support VCS and Xcelium for coverage UNR")    
```

Comment:
Not sure how the lint didn't catch this, but there is extra whitespace at the end of the line.